janus_core/calculations/phonons.py

```diff
@@ -232,8 +238,8 @@ def __init__(
 
         # Ensure supercell is a valid list
         self.supercell = [supercell] * 3 if isinstance(supercell, int) else supercell
-        if len(self.supercell) != 3:
-            raise ValueError("`supercell` must be an integer, or list of length 3")
+        if len(self.supercell) not in [3, 9]:
+            raise ValueError("`supercell` must be an integer, or list of length 3 or 9")
```

Comment:
you need to be more informative, 9 in what order for matrix.

Reply:
The goal was to reproduce the [phonopy behaviour](https://phonopy.github.io/phonopy/setting-tags.html#dimension-tag), so first three values are the first row, etc. Should I write that explicitly into the documentation, or would you prefer it elsewhere?

Reply:
In some ways "following Phonopy convention" seems _more_ informative than listing array elements.

Reply:
do not assume that anyone will know what is the phonopy convention... just list it.

Reply:
I have changed the documentation and error messages; is the current version OK, or do I need to be more detailed/write it in another way?

Reply:
@alinelena do we actually want the `ValueError` to be quite so detailed?

I agree when we do specify the form of the inputs, we should be clear in our code and not just reference external codes e.g. Phonopy, but to me, something like the order the matrix elements makes more sense sticking to the docstring, otherwise we end up with a lot of duplication.

Maybe it makes sense though, given that it's probably the most likely thing people might trip up on